service-app/internal/api/queue_handler.go

```diff
@@ -26,22 +27,12 @@ func (c *IndexController) processQueue(ctx context.Context, scannedCaseResponse
 
 			attchResp, decodedXML, docErr := service.AttachDocuments(ctx, scannedCaseResponse)
 			if docErr != nil {
-				c.logger.ErrorWithContext(ctx, "Failed to attach document", map[string]any{
-					"set_uid":       scannedCaseResponse.UID,
-					"document_type": originalDoc.Type,
-					"error":         docErr.Error(),
-				})
 				return docErr
```

Comment:
would it be worth having these two use `fmt.Errorf` so you don't lose the attach/persist?